netci.groovy

```diff
@@ -1064,11 +1064,15 @@ combinedScenarios.each { scenario ->
 
                                     // Up the timeout for arm64 jobs.
                                     Utilities.setJobTimeout(newJob, 240);
-                                    buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${architecture} /toolset_dir C:\\ats"
 
-                                    // Debug runs take too long to run.
-                                    if (lowerConfiguration != "debug") {
-                                       buildCommands += "Z:\\arm64\\common\\scripts\\arm64PostBuild.cmd %WORKSPACE% ${architecture} ${lowerConfiguration}"
+                                    // Debug runs take too long to run. So build job only.
+                                    if (lowerConfiguration == "debug") {
+                                       buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${architecture} /toolset_dir C:\\ats"
+                                    }
+                                    else {
+                                       buildCommands += "set __TestIntermediateDir=int&&build.cmd skiptests ${lowerConfiguration} ${architecture} /toolset_dir C:\\ats"
+                                       // Test build and run are launched together.
+                                       buildCommands += "Z:\\arm64\\common\\scripts\\arm64PostLauncher.cmd %WORKSPACE% ${architecture} ${lowerConfiguration}"
```

Comment:
I could checkin only those files, but currently they won't be consumed directly and also I need to sync them in source tree whenever changes are needed. As you know, there are actually more scripts for this arm64 run in the lab. I think once "arm64PostLauncher" just copy all the scripts from the source tree and then start the subsequent process that is currently doing, then we can open their sources that are synced automatically. And also currently there is some hard-coded logic that should be credential, which means first of all I should sanitize the source.
So I think we should follow this up later all together. Feel free to open an GH issue.